lib/sycamore/sycamore/tests/unit/transforms/test_extract_entity.py

```diff
@@ -0,0 +1,29 @@
+from sycamore.transforms.extract_entity import OpenAIEntityExtractor
+from sycamore.data import Document, Element
+from sycamore.llms.prompts.default_prompts import EntityExtractorFewShotGuidancePrompt
+
+
+class FakeLLM:
+    def __init__(self):
+        pass
+
+    def generate(self, prompt_kwargs):
+        assert isinstance(prompt_kwargs["prompt"], EntityExtractorFewShotGuidancePrompt)
+        assert prompt_kwargs["entity"] == "title"
+        assert prompt_kwargs["query"] == "ELEMENT 1: ExampleTitle\nELEMENT 2: Footer\n"
+        assert prompt_kwargs["examples"] == "An Example Template"
+        return "ExampleTitle"
+
+
+def test_extract_with_template():
```

Comment:
can you consolidate these in https://github.com/aryn-ai/sycamore/blob/main/lib/sycamore/sycamore/tests/unit/transforms/test_entity_extraction.py

Reply:
Done, also renamed the file since the test file name should match the file being tested.